config/database.php

```diff
@@ -50,7 +50,7 @@
             'collation' => 'utf8mb4_unicode_ci',    // default: utf8mb4_unicode_ci , Laravel 5.3: utf8_unicode_ci
             'prefix' => '',
             'strict' => false,  // default: true
-            'engine' => null,
+            'engine' => 'InnoDB ROW_FORMAT=DYNAMIC',
```

Comment:
Whats the matter with this line?

Reply:
correct commands for using the current mariadb
will avoid such errors: https://laracasts.com/discuss/channels/eloquent/migrations-and-table-options-row-format
i got that local on my machine